### ISSUE TITLE
Convert spoiler handling to CSS.

### ISSUE THREAD

**Issue by RePod** [spootTheLousy/saguaro#37]
Currently, spoilers work with the following attributes attached to their elements:

``` HTML
onmouseout="this.style.color=this.style.backgroundColor='#000'"
onmouseover="this.style.color='#FFF';"
style="color: rgb(0, 0, 0); background: rgb(0, 0, 0) none repeat scroll 0% 0%;"
```

However, this can be done cleaner using CSS:

``` CSS
.spoiler       { color: #000; background: #000; }
.spoiler:hover { color: #FFF; }
```

**Advantages:**
- Safer.
- Faster.
- Cleaner.
- Scalable.
- Smaller.
  - In general.
  - But also if there are several spoilers on the page this would cut off a ton of bytes (and a few in the code).
- Able to be themed.

**Disadvantages:**
- If it's not themed then it won't be spoiled.
  - Could be dealt with by a "required" CSS file that doesn't need to be modified by users and exists only to keep certain things (like spoilers) working.
  - Alternatively, don't care and blame theme creators for not supporting spoilers.

**Comment by spootTheLousy** [comment]
Sounds like a good way to clean it up a bit better. I'd say just add the spoiler stuff to the included csses and such. Let people with custom themes add the new stuff in.

**Comment by Apogate** [comment]
Fixed in bf5213f2e0b3cdd9fd44a1f9d166695190b28e0d
